### PR TITLE
Fixed docs #778, #779, #780

### PR DESCRIPTION
**Pull request by bocajnotnef**
Resolves dead link #778
Resolves naming inconsistency #779

- [x] Is it mergable?
- [x] Did it pass the tests?
- [x] If it introduces new functionality in scripts/ is it tested?
  Check for code coverage with `make clean diff-cover`
- [x] Is it well formatted? Look at `make pep8`, `make diff_pylint_report`,
  `make cppcheck`, and `make doc` output. Use `make format` and manual
  fixing as needed.
- [x] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [x] Is it documented in the ChangeLog?
  http://en.wikipedia.org/wiki/Changelog#Format
- [x] Was a spellchecker run on the source code and documentation after
  changes were made?

### CODE REVIEW

**Comment by bocajnotnef** [comment]
Test this please

**Comment by bocajnotnef** [comment]
Does not in fact fix #780.

**Comment by bocajnotnef** [comment]
@mr-c

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,8 @@
+2015-02-19  Jacob Fenton  <bocajnotnef@gmail.com>
+
+   * doc/introduction.txt, doc/user/choosing-table-sizes.txt: Updated docs to
+   refernce correct links and function arguments.
```
Comment (ctb) [inline ChangeLog]: "reference"

**Comment by ctb** [comment]
Please use Count-Min Sketch as in the Cormode paper and http://www.ncbi.nlm.nih.gov/pubmed/25062443, and add that as a reference.

**Comment by bocajnotnef** [comment]
Define "use" in this context, please; am I simply changing the paper referenced?

**Comment by ctb** [comment]
Pls use the term "Count-Min Sketch" instead of CountMin or MinCount or whatnot... tnx!

**Comment by bocajnotnef** [comment]
retest this please

**Comment by bocajnotnef** [comment]
@ctb CR?

File: ChangeLog [hunk 2]
```diff
@@ -11,6 +16,7 @@
    column headers.
    * tests/test_scripts.py: add coverage for the new --csv option in
    abundance-dist.py
+>>>>>>> 43e2217d4078d9c42f1e10df580e074a44c0efaa
```
Comment (mr-c) [inline ChangeLog]: ? (and lines 72 and 162)

File: doc/user/choosing-table-sizes.txt [hunk 3]
```diff
@@ -60,7 +60,7 @@ The real full version
 
 khmer's scripts, at their heart, represents k-mers in a very memory
 efficient way by taking advantage of two data structures, `Bloom
-filters <http://en.wikipedia.org/wiki/Bloom_filter>`__ and `CountMin
+filters <http://en.wikipedia.org/wiki/Bloom_filter>`__ and `Count-Min
 Sketches <https://sites.google.com/site/countminsketch/>`__, that are
```
Comment (ctb) [inline doc/user/choosing-table-sizes.txt]: http://en.wikipedia.org/wiki/Count%E2%80%93min_sketch here instead of google site - tnx.

**Comment by bocajnotnef** [comment]
@mr-c @ctb Should be clean, now

**Comment by ctb** [comment]
Thanks!

**Pull request merged**